katas/content/KatasLibrary.qs

```diff
@@ -6,43 +6,21 @@ namespace Microsoft.Quantum.Katas {
     open Microsoft.Quantum.Diagnostics;
     open Microsoft.Quantum.Math;
     open Microsoft.Quantum.Random;
-
-    /// # Summary
-    /// Given two operations, checks whether they act identically for all input states.
-    /// This operation is implemented by using the Choi–Jamiołkowski isomorphism.
-    operation CheckOperationsEquivalence(
-        op : (Qubit[] => Unit is Adj + Ctl),
-        reference : (Qubit[] => Unit is Adj + Ctl),
-        inputSize : Int)
-    : Bool {
-        Fact(inputSize > 0, "`inputSize` must be positive");
-        use (control, target) = (Qubit[inputSize], Qubit[inputSize]);
-        within {
-            EntangleRegisters(control, target);
-        }
-        apply {
-            op(target);
-            Adjoint reference(target);
-        }
-
-        let areEquivalent = CheckAllZero(control + target);
-        ResetAll(control + target);
-        areEquivalent
-    }
+    open Microsoft.Quantum.Diagnostics;
 
     /// # Summary
     /// Given two operations, checks whether they act identically (including global phase) for all input states.
     /// This is done through controlled versions of the operations instead of plain ones which convert the global phase
     /// into a relative phase that can be detected.
-    operation CheckOperationsEquivalenceStrict(
+    operation CheckOperationsAreEqualStrict(
```

Comment:
Now that I think about it, we should probably change the order of arguments here as well for the inputSize to go first, to match the library `CheckOperationsAreEqual`. It can be jarring if two similar operations use different order of arguments, even if they technically come from different libraries